404.html

```diff
@@ -0,0 +1,21 @@
+---
+layout: default
+---
+
+<div class="row">
+    <div class="container my-5 py-5">
+        <div class="row">
+            <div class="offset-lg-2 col-lg-7 mt-md-4">
```

Comment:
After https://github.com/compilerla/compiler.la/pull/134 and https://github.com/compilerla/compiler.la/pull/135 are merged, this line can be changed to `<div class="offset-lg-1 col-lg-7">`